context-free.md

```diff
@@ -1,22 +1,26 @@
 # Context Free
 
 ##  Who is this programming language for?
+The language is for people with an interest and affinity for art, programming, and grammars.  It provides a convenient combination of each of those things, allowing anyone with a sense for at least one of them to pick it up rather easily.
 
 
 ## What is easy to do in this language? Why is it easy?
+It is very easy to define something recursively, by simply defining your grammar and then letting the rendering engine take over.  It's easy because it requires very little manual entry by the artist.  Simple shapes and variations of simple shapes are also easy.
 
 
 ## What is hard to do in this language? Why is it hard?
+The non-recursive sections of the image were hard - it required a lot more manual fiddling.  Additionally, I had a really hard time with overlapping sections as well as shape parameters (which I gave up on) and loops (which I mostly replaced with recursion).  
 
```

Comment:
It's almost like the language is pushing you toward one designed choice rather than another (i.e. recursion over loops). :)